### PR TITLE
Add bug report link.

### PR DESCRIPTION
**Pull request by hotchemi**
## What

Just add bug report link to the github.

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/res/layout/fragment_settings.xml [hunk 1]
```diff
@@ -65,6 +65,27 @@
...
+                android:id="@+id/txt_bugreport"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:lines="1"
+                android:text="@string/settings_bugreport"
+                android:textColor="@color/black"
+                android:textSize="@dimen/text" />
+
+        </RelativeLayout>
```
Comment (konifar) [inline app/src/main/res/layout/fragment_settings.xml]: I think only TextView is better.

```xml
<TextView
    android:id="@+id/txt_bugreport"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:background="@drawable/clickable"
    android:padding="@dimen/spacing"
    android:clickable="true"
    android:ellipsize="end"
    android:lines="1"
    android:text="@string/settings_bugreport"
    android:textColor="@color/black"
    android:textSize="@dimen/text" />
```

File: app/src/main/res/values/strings_urls.xml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="about_inquiry_url">https://docs.google.com/forms/d/1fpxZOqprZyzBgMBb7HCCrq538oOzfzWkOMpItRqF-ro/viewform</string>
+    <string name="bug_report_url">https://github.com/konifar/droidkaigi2016/issues/new?title=%5BBug%20Report%5D&amp;body=%23%20Problem%0A%0A%23%20Solution</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_urls.xml]: Nice :+1:

File: app/src/main/res/layout/fragment_settings.xml (hunk 1 above)
Comment (hotchemi) [inline app/src/main/res/layout/fragment_settings.xml]: Fixed!

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java [hunk 3]
```diff
@@ -53,6 +56,7 @@ public void onAttach(Context context) {
     private void initView() {
         binding.txtLanguage.setText(AppUtil.getCurrentLanguage(getActivity()));
         binding.languageSettingsContainer.setOnClickListener(v -> showLanguagesDialog());
+        binding.bugreportSettingsContainer.setOnClickListener(v -> showBugReport());
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java]: Is `binding.txtBugreport` right?

Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java]: Super sorry :cry:

**Comment by konifar** [comment]
LGTM!!

Thanks for your quick development :+1:

**Comment by hotchemi** [comment]
@konifar Thx for your report! Btw, could u add me to collaborator list?

**Comment by konifar** [comment]
@hotchemi I added you :)

**Comment by hotchemi** [comment]
:+1:

**Comment by konifar** [comment]
You can merge this :clap:

**Pull request merged**